src/hc-06.cpp

```diff
@@ -0,0 +1,65 @@
+// ==========================================================================
+//
+// File      : hc-06.cpp
+// Part of   : C++ library for using the hc-06 bluetooth module.
+//
+// ==========================================================================
+
+// this file contains Doxygen lines
```

Comment:
Incorrect use of Doxygen, please read the wiki for the correct Doxygen standard.